### DISCUSSION TITLE
Custom resolutions not loading?

### DISCUSSION THREAD

**Discussion by MarkBowenPiano** [lllyasviel/Fooocus#3339] [Q&A] [answered]
Hi there,

Sorry this is my first time posting on this site so hoping I'm not doing anything wrong anywhere.
I've just installed Fooocus using Pionkio on my Mac Mini M2 Pro and wanted to use some different resolutions / aspect ratios alongside the pre-installed ones.

I added this to my config.tx file and saved the file then loaded up Pinokio and Fooocus 👍 

```
    "available_aspect_ratios": [
        "704*1408",
        "704*1344",
        "768*1344",
        "768*1280",
        "832*1216",
        "832*1152",
        "896*1152",
        "896*1088",
        "960*1088",
        "960*1024",
        "1024*1024",
        "1024*960",
        "1088*960",
        "1088*896",
        "1152*896",
        "1152*832",
        "1216*832",
        "1280*768",
        "1344*768",
        "1344*704",
        "1408*704",
        "1472*704",
        "1536*640",
        "1600*640",
        "1664*576",
        "2048*1152"
    ],
    "default_aspect_ratio": "2048*1152",

```
The specific one being 2048*1152 that I would like added. Loading up Pinokio and Fooocus after saving that to the config.txt file doesn't do anything at all. When I load it all up I still get all the exact same resolutions and my 'new' resolution is nowhere to be seen :-(
I thought maybe the app could only show so many resolutions so I edited the last resolution to be the one I wanted but all the same standard resolutions that come pre-installed show up. No other config changes work either :-(

I know it must be something silly that I'm missing somewhere but not really sure where to look?

Any help with this would be greatly appreciated.

Many thanks,

Mark

**Comment by mashb1t** [comment]
LGTM  Check out the console logs to see if there's another error in your config file.

**Reply by MarkBowenPiano** [reply]
Thanks. How do I get to the console log from inside Pinokio? Do you possibly know as I'm totally new to all of this.

Many thanks,

Mark

**Reply by mashb1t** [reply]
Sorry, no idea, also never used Pinokio and there is no official support for it.

**Reply by MarkBowenPiano** [reply]
Okay nevermind. Thank you for the help though it is very much appreciated. I'll see what I can find out by going through the entire application. Hopefully I'll find something somewhere.

Best wishes,

Mark

**Comment by MarkBowenPiano** [comment, marked as answer]
Actually thank you to mashb1t for your help. I managed to find the log file in the end it was at :

**Users/myusername/pinokio/logs/stdout.txt**

and it pointed me to the issue. I'd never edited a config file before and had copied the code from online. They had a comma after the last bit of information before the end of the code which shouldn't have been there.
Deleted that and all is working now. Thanks for all the help.
